CHANGELOG.md

```diff
@@ -1,5 +1,74 @@
+- selfcheck: add column for comments [\#2744](https://github.com/rero/rero-ils/pull/2744) [[f: circulation](https://github.com/rero/rero-ils/labels/f:%20circulation)] [[f: data migration](https://github.com/rero/rero-ils/labels/f:%20data%20migration)] ([lauren-d](https://github.com/lauren-d))
+
+**Fixed bugs:**
+
+- Field ` reproductionOf` has wrong label in the document public view [\#2801](https://github.com/rero/rero-ils/issues/2801) [[f: public ui](https://github.com/rero/rero-ils/labels/f:%20public%20ui)]
+- The due date of a loan is displayed with a delay of one day in the UI [\#2748](https://github.com/rero/rero-ils/issues/2748) [[f: circulation](https://github.com/rero/rero-ils/labels/f:%20circulation)] [[f: professional ui](https://github.com/rero/rero-ils/labels/f:%20professional%20ui)] [[f: public ui](https://github.com/rero/rero-ils/labels/f:%20public%20ui)]
+- A late renewal does not effectively generate the overdue preview fees [\#2741](https://github.com/rero/rero-ils/issues/2741) [[f: circulation](https://github.com/rero/rero-ils/labels/f:%20circulation)]
+- Item tasks: temporary locations are not cleaned correctly [\#2694](https://github.com/rero/rero-ils/issues/2694) [[f: activity-logs](https://github.com/rero/rero-ils/labels/f:%20activity-logs)]
+- "Show more" button doesn't work in public interface [\#2691](https://github.com/rero/rero-ils/issues/2691) [[f: public ui](https://github.com/rero/rero-ils/labels/f:%20public%20ui)]
+- Add fixtures: accounts, library acquisition settings, order and order lines [\#2687](https://github.com/rero/rero-ils/issues/2687)
+- ProvisionActivity field : wrong separator for agents  [\#2670](https://github.com/rero/rero-ils/issues/2670) [[f: editor](https://github.com/rero/rero-ils/labels/f:%20editor)]
+- Item number displayed on holdings for serial and standard items [\#2494](https://github.com/rero/rero-ils/issues/2494) [[f: public ui](https://github.com/rero/rero-ils/labels/f:%20public%20ui)] [[f: serials](https://github.com/rero/rero-ils/labels/f:%20serials)]
+- Cancellation of a request leads in some cases to the wrong item status [\#2454](https://github.com/rero/rero-ils/issues/2454) [[f: circulation](https://github.com/rero/rero-ils/labels/f:%20circulation)]
+- In the notification "Prêt impossible car l’exemplaire est déjà prêté" precise the document is on loan to another patron [\#2401](https://github.com/rero/rero-ils/issues/2401) [[f: circulation](https://github.com/rero/rero-ils/labels/f:%20circulation)] [[f: professional ui](https://github.com/rero/rero-ils/labels/f:%20professional%20ui)]
+- Item location and category update when parents holdings is changed. [\#1436](https://github.com/rero/rero-ils/issues/1436) [[f: data](https://github.com/rero/rero-ils/labels/f:%20data)] [[f: serials](https://github.com/rero/rero-ils/labels/f:%20serials)]
+- dojson: fix work access point and title [\#2824](https://github.com/rero/rero-ils/pull/2824) ([rerowep](https://github.com/rerowep))
```

Comment:
From there we have pull requests in the list. Why are they not in the next section?